### PR TITLE
Fix tests

### PR DESCRIPTION
**Pull request by Xorima**
### Description

Fixes tests ... I hope

### Issues Resolved

[List any existing issues this PR resolves]

### Contribution Check List

- [ ] All tests pass.
- [ ] New functionality includes testing.
- [ ] New functionality has been documented in the README if applicable

### CODE REVIEW

**Comment by marcosvm** [comment]
@Xorima I can help with this PR.

**Comment by Xorima** [comment]
Hey @marcosvm 

I'd love the help, if you are not already a member of sous-chefs please request to join here: <https://github.com/sous-chefs/terraform-github-org> 

Drop by the #sous-chefs channel as well in https://community-slack.chef.io/ and say hi, 

I just want to see this green so we can start on some new features and bug requests, hard to merge things as there is a much larger chance I am breaking something else

**Comment by marcosvm** [comment]
@Xorima I can't reproduce the ```STDERR: /usr/sbin/mysqld: Can't read dir of '/etc/mysql/conf.d/' (Errcode: 13 - Permission denied)``` we're seeing all over the CI failed builds. I tried a MacBook and inside a VM running Ubuntu 18.04, in both cases chef converges OK. 

Do you have any recommendation of how to reproduce this error?

**Comment by Xorima** [comment]
Hey, are you using dokken or full vagrant?

**Comment by marcosvm** [comment]
I'm using dokken, I think CI is using dokken. I'll try vagrant too.

File: recipes/configure_server.rb [hunk 1]
```diff
@@ -142,6 +145,11 @@
                    end
 end
 
+# Reload ohai for version info on packages before creating the cluster
+ohai 'reload' do
+  action :reload
+end
+
```
Comment (devsibwarra) [inline recipes/configure_server.rb]: This resource name seems generic and a fair chance of naming overlap.  Should this be a unique name like `reload_percona`?

**Pull request closed without merging**